<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LambdaSnail::Juno::charts::LSExpenseChartsWidget</class>
 <widget class="QWidget" name="LambdaSnail::Juno::charts::LSExpenseChartsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LSExpenseChartsWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="mainChartFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
